src/welcome/Welcome.tsx

```diff
@@ -1,18 +1,51 @@
-import "./Welcome.scss";
 import React from "react";
+import { makeStyles, createStyles, Theme, Typography } from "@material-ui/core";
 import Login from "../components/Login";
-
 import image from "../assets/welcome-screen.png";
+import { lavenderBlush } from "theme";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
```

Comment:
Query: is there any way to have these styles in a different file? Would just be a bit neater going forward if there was a separation between styles and everything else

Reply:
This is the way Material UI does styles and how it's done with other libraries such as styled-components.

I think in React & Vue it's fairly common to keep everything associated with a file (except tests) encapsulated within a file because it makes the codebase look simpler, means if you change a file path you don't, in turn, have to change loads more file paths and because those styles aren't used anywhere else, it makes sense to keep them private within that file.

Also, if a file is too large with the styles & tsx, it might be an indication that it needs to be broken down into separate components. 

Have a think about those points, but if your opinion stays the same I'm happy to look into extracting this, as it should be easy but I'd need to confirm.

Reply:
Yeah makes sense, happy to go forward with that setup then, just wanted to ask to see if it was possible